front/src/applications/stdcm/components/StdcmResults/StdcmSimulationReportSheet.tsx

```diff
@@ -117,6 +121,7 @@ const StdcmSimulationReportSheet = ({
           stdcmLinkedTrains={stdcmLinkedTrains}
           routeTableRows={routeOperationalPoints}
         />
+        <SchedulesToDuplicate refSchedules={refSchedules} />
```

Comment:
Hum Typescript does not break even though the given type does not match the expected type here...  It's weird